discovery_engine_core/bindings/src/lib.rs

```diff
@@ -40,48 +40,32 @@ use std::path::Path;
 use itertools::Itertools;
 use xayn_discovery_engine_core::Engine;
 
-#[allow(unsafe_code)]
-#[no_mangle]
-pub extern "C" fn cfg_feature_storage() -> u8 {
-    u8::from(cfg!(feature = "storage"))
-}
-
 #[async_bindgen::api(
     use uuid::Uuid;
 
-    use xayn_discovery_engine_ai::Embedding;
     use xayn_discovery_engine_core::{
-        document::{Document, HistoricDocument, TimeSpent, TrendingTopic, UserReacted, WeightedSource},
-        storage2::DartMigrationData,
+        document::{TimeSpent, UserReacted},
         InitConfig,
     };
     use xayn_discovery_engine_providers::Market;
 
-    use crate::types::{engine::{SharedEngine, InitializationResult}, search::Search};
+    use crate::types::{
+        document::Document,
+        engine::{InitializationResult, SharedEngine},
+        search::Search,
+        trending_topic::TrendingTopic,
+    };
 )]
 impl XaynDiscoveryEngineAsyncFfi {
     /// Initializes the engine.
-    #[allow(clippy::box_collection)]
-    pub async fn initialize(
-        config: Box<InitConfig>,
-        state: Option<Box<Vec<u8>>>,
-        history: Box<Vec<HistoricDocument>>,
-        sources: Box<Vec<WeightedSource>>,
-        dart_migration_data: Option<Box<DartMigrationData>>,
```

Comment:
do we use migration data somewhere else?

Reply:
no, only in the `Engine::from_config()` constructor.

Reply:
I saw it was moved into the config 👍